File: java/src/main/java/com/languageweaver/sdk/samples/dictionaries/DictionaryService.java
package com.languageweaver.sdk.samples.dictionaries;

import com.languageweaver.sdk.LanguageWeaverClient;
import com.languageweaver.sdk.SdkFactory;
import com.languageweaver.sdk.configurations.ClientConfiguration;
import com.languageweaver.sdk.translate.result.DictionariesResult;

public class DictionaryService {

    public static void main(String[] args) throws Exception {
        try (LanguageWeaverClient lwClient = new SdkFactory().getLanguageWeaverClient(new ClientConfiguration())) {
            final DictionariesResult dictionariesResult = lwClient.getDictionaries();

            if (dictionariesResult.getDictionaries() != null) {
                dictionariesResult.getDictionaries()
                        .forEach(dictionary -> System.out.println(dictionary.getId()));
            }
        }
    }
}

File: javascript/browser/src/samples/dictionaries/DictionaryService.js
import SdkFactory, {ClientConfiguration, CredentialsConfiguration} from "@language-weaver/lw-sdk-js";

export const getDictionaries = async (clientId, clientSecret) => {
    try {
        const clientConfiguration = new ClientConfiguration();
        clientConfiguration.credentialsConfiguration = new CredentialsConfiguration(clientId, clientSecret);
        const lwClient = await SdkFactory.getLanguageWeaverClient(clientConfiguration);
        const dictionariesResult = await lwClient.getDictionaries(1);
        console.log(dictionariesResult.dictionaries);
    } catch (e) {
        console.log(e);
    }
}